noxfile.py

```diff
@@ -1129,3 +1129,26 @@ def docs_man(session, compress, update):
     if compress:
         session.run("tar", "-cJvf", "man-archive.tar.xz", "_build/man", external=True)
     os.chdir("..")
+
+
+@nox.session(name="changelog", python="3")
+@nox.parametrize("draft", [False, True])
+def changelog(session, draft):
+    """
+    Generate salt's changelog
+    """
+
+    pydir = _get_pydir(session)
+    requirements_file = "requirements/static/changelog.in"
+    distro_constraints = [
+        "requirements/static/{}/changelog.txt".format(_get_pydir(session))
```

Comment:
Use `pydir` defined above?

Reply:
i removed `pydir` instead since i dont use it anywhere else. I did update the calls from the tests above that do use `pydir` but not with `distro_constraints`. I also updated the format call. thanks